src/overlays/actors/ovl_En_Baba/z_en_baba.c

```diff
@@ -128,142 +149,145 @@ static TrackOptionsSet sTrackOptions = {
     { 0x1770, 4, 1, 6 },
 };
 
-s32 func_80BA8820(EnBaba* this, PlayState* play) {
-    this->unk_144 = (EnOssan*)SubS_FindActor(play, &this->unk_144->actor, ACTORCAT_NPC, ACTOR_EN_OSSAN);
+s32 EnBaba_FindBombShopkeeper(EnBaba* this, PlayState* play) {
+    //! The bomb shopkeeper is an EnSob1, but initalizes itself with id `ACTOR_EN_OSSAN`
```

Comment:
Does this deserve a @bug in front of it?

Reply:
I don't really consider it a bug. It is more an unsafe design choice that I want to bring attention to.